thirdeye/thirdeye-frontend/app/pods/components/anomaly-stats-block/component.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * Component for anomaly performance statistics block
+ * @module components/anomaly-stats-block
+ * TODO: list properties, example
+ * @exports anomaly-stats-block
+ * @author smcclung
+ */
+
+import Component from '@ember/component';
+
+export default Component.extend({
+  classNames: ['te-horizontal-cards__container']
```

Comment:
nit: Since it's a component, shall we just set the BEM block to `anomaly-stats` instead of the parent `te-horizontal-cards`?